crates/bevy_app/src/app.rs

```diff
@@ -62,6 +68,7 @@ impl App {
         self.schedule.run(&mut self.world);
     }
 
+    /// Calls the runner function of the `App`.
```

Comment:
```suggestion
    /// Calls the runner function of the `App`, executing its [`Schedule`] to run systems
```

Reply:
Thanks. I'll see if there are other occurrences of this.